src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -438,6 +446,13 @@ private function registerGraphqlConfiguration(ContainerBuilder $container, array
         $container->setParameter('api_platform.graphql.graphiql.enabled', $config['graphql']['graphiql']['enabled']);
 
         $loader->load('graphql.xml');
+
+        if (isset($bundles['TwigBundle'])) {
```

Comment:
It looks wrong to me. It means that you can get GraphQL support without Twig.

As for Swagger UI, you should throw if `api_platform.graphql.graphiql.enabled` is `true` but Twig isn't installed.